ads/vendors/jixie.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```

Comment:
nit: 2021

Reply:
Oooops, changed this and the jixie.md too. Thanks!
Also I would like to check with you: in the _config.js file I had both these specified: 
    clientIdScope: '__jxamp',
    clientIdCookieName: '_jx',
It is fine, right?
The CookieName one has priority and might be available when in non-amp-cache scenario.
And if and when cookie is available, then it will be the one given (in the 'context' object)
if not, then the amp-managed clientid (as specified by the scope) will be passed in.
Did I understand it (from the public doc) right?

Reply:
This is correct. The cookie name will default to the `clientIdScope` if you prefer to use a single value.

Reply:
Hi thanks for your note!
A bit anxious for our adaptor to be included into the runtime but 
it seems the pull request is still blocking on something (some checking)
Can you kindly help me check? I am quite new to this kind of collaborative things.
Great thanks!!

Reply:
Thanks a lot for all the answers. Looking for a place I can press "reply" and can be more sure the thing will reach you.
Anyway - so I made some changes based on the cirlceCI complaints ! And I think I put it to request re-review.